src/js/containers/search/table/ResultsTableContainer.jsx

```diff
@@ -438,7 +438,7 @@ const ResultsTableContainer = (props) => {
         if (SearchHelper.isSearchHashReady(location)) {
             pickDefaultTab();
         }
-    }, [location]);
+    }, []);
```

Comment:
you should add the comment to disable exhaustive deps here, to make sure location isn't auto-added again; and you now have two useEffects with [] as a dep so you can combine them

Reply:
done